backend/attendance-service/src/main/java/org/egov/util/StaffPermissionServiceUtil.java

```diff
@@ -0,0 +1,26 @@
+package org.egov.util;
+
+
+import digit.models.coremodels.AuditDetails;
+import org.egov.web.models.StaffPermission;
+import org.springframework.stereotype.Component;
+
+@Component
+public class StaffPermissionServiceUtil {
+
+    /**
+     * Method to return auditDetails for create/update flows
+     *
+     * @param by
+     * @param isCreate
+     * @return AuditDetails
+     */
+    public AuditDetails getAuditDetails(String by, StaffPermission staffPermission, Boolean isCreate) {
```

Comment:
Move this to a common util file, don't keep it as part of `StaffPermissionUtil`.

Reply:
File deleted. Using the common AttendanceServiceUtil file now.